### ISSUE TITLE
Недостатки в работе с закладками у новой версии

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#689]
```
What steps will reproduce the problem?
Пытаюсь добавить закладку, как в 
предыдущей версии программы

What is the expected output?
По тапу закладка сейчас не добавляется. 
Пробую добавить команду "Добавить 
закладку" через настройку нажатий (зон 
касания), однако касающиеся закладок 
команды просто не запоминаются. 
Единственный способ, которым я могу 
добавить закладку - нажать кнопку 
"Добавить" под перечнем закладок. Но это не 
удобно, так как при этом правая панель 
занимает половину экрана, и не видно текста.

What do you see instead?
Вот видео, иллюстрирующее проблему: 
http://youtu.be/jAI1guk-UqM

What version of the product are you using?
Последняя версия

On what operating system?
Android 4.0.1

What device are you using?
Huawei MediaPad 10FHD

Please provide any additional information below.
Отличная программа! Спасибо Вам!
```

Original issue reported on code.google.com by `ver.v...@gmail.com` on 7 Jan 2014 at 12:38

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1.1
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 9 Jan 2014 at 8:20
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
С закладками порядок, спасибо что 
исправили!

Не совсем правда понятно, как пользоваться 
заметками и пометками. Если вешаю "Заметка" 
на нажатие, то при тапе всё равно 
открывается "Добавить закладку". Если вешаю 
"Добавить пометку", то при тапе вообще 
ничего не происходит. Проверьте пожалуйста.
```

Original comment by `ver.v...@gmail.com` on 10 Jan 2014 at 1:30
